vscode-extension/src/aiConfigEditor.ts

```diff
@@ -340,12 +341,23 @@ export class AIConfigEditorProvider implements vscode.CustomTextEditorProvider {
 
           const message = notification.message;
 
-          // Notification supports 'details' for modal only. For now, just show title
-          // in notification toast and full message in output channel.
-          const notificationAction = await notificationFn(
-            notification.title,
-            message ? "Details" : undefined
-          );
+          let notificationAction;
+          // TODO: Create a constant value somewhere in lastmile-utils to
+          // centralize string error message for missing API key. This
+          // logic is defined in https://github.com/lastmile-ai/aiconfig/blob/33fb852854d0bd64b8ddb4e52320112782008b99/python/src/aiconfig/util/config_utils.py#L41
+          if (message.includes("Missing API key")) {
+            notificationAction = await notificationFn(
+              "Looks like you're missing an API key, please set it in your env variables",

```

Comment:
One thing I found out recently is that vscode has MarkdownString which can be used in some vscode components. If it works, would be great to use to show documentation link here as well:

```
  const message = new vscode.MarkdownString();
  message.appendMarkdown(
    "Looks like you're missing an API key. Please set it in your <a href='https://aiconfig.lastmileai.dev/docs/getting-started/#setup-your-api-keys' target='_blank'>.env variables</a>"
  );
  message.isTrusted = true; // allow links to be clickable -- not sure if needed
```

Reply:
I tried getting this to work but wasn't able to for diagnostics messaging. Seems like it's still an open issue: https://github.com/microsoft/vscode/issues/54272